static/app/components/footer.tsx

```diff
@@ -39,7 +47,14 @@ function BaseFooter({className}: Props) {
         {privacyUrl && <FooterLink href={privacyUrl}>{t('Privacy Policy')}</FooterLink>}
         {termsUrl && <FooterLink href={termsUrl}>{t('Terms of Use')}</FooterLink>}
       </LeftLinks>
-      <LogoLink />
+      <SentryLogoLink href="https://sentry.io/welcome/" tabIndex={-1}>
+        {!isSelfHosted &&
+        !!(organization.features ?? []).includes('sentry-pride-logo-footer') ? (
+          <SentryPrideLogoHook size="lg" />
+        ) : (
+          <IconSentry size="lg" />
+        )}
+      </SentryLogoLink>
```

Comment:
```suggestion
      <SentryLogoLink href="https://sentry.io/welcome/" tabIndex={-1}>
          <SentryLogoHook size="lg" />
      </SentryLogoLink>
```
No need to check isSelfHosted as the hook will default to the regular logo. I would move the feature flag check to getsentry.

Reply:
can confirm that we don't need to tch `isSelfHosted` ..thanks Matej!